search_events_app/views.py

```diff
@@ -20,21 +20,11 @@ def get_queryset(self):
         return StateManager.get_last_searched_events()
 
     def get_context_data(self, **kwargs):
-        countries = Country.objects.all()
-        languages = Language.objects.order_by('name')
-
         context = super().get_context_data(**kwargs)
-        context['countries'] = [
-            {
-                'alpha2Code': country.alpha_2_code,
-                'name': country.name
-            } for country in countries
-        ]
-
-        context['languages'] = [
-            {
-                'code': language.code,
-                'name': language.name
-            } for language in languages
-        ]
+
+        classes = [Country, Language]
```

Comment:
I not sure whats mean this line... could you explain me?

Reply:
We have created a class called GetContextMixin, which is inherited by Country and Language, and will be inherited by Feature in the future. So we add more classes to that 'classes' array to fill the context variable